api/app/utils/daily_pdf_gen.py

```diff
@@ -0,0 +1,44 @@
+"""Generate a daily PDF"""
+from typing import List, Mapping
+import pdfkit
+from jinja2 import Template
+
+from app.schemas.hfi_calc import FireCentre, HFIResultResponse, PlanningArea, WeatherStation
+from app.utils.pdf_data_formatter import response_2_daily_jinja_format
+
+daily_template_path = "api/app/utils/daily_template.html"
```

Comment:
Depending on where your code is execute from this path may not work. You have to do something like use the `__file__` variable to get the correct path. e.g. `os.path.join(os.path.dirname(__file__), 'daily_template.html')`

Reply:
Awesome thanks